core/common/src/main/java/alluxio/metrics/WorkerMetrics.java

```diff
@@ -15,6 +15,34 @@
  * Metrics of an Alluxio worker.
  */
 public final class WorkerMetrics {
+  /**
+   * The list of under filesystem operations.
+   */
+  public enum UfsOps {
+    Close,
+    ConnectFromMaster,
+    ConnectFromWorker,
+    Create,
+    DeleteDirectory,
+    DeleteFile,
+    Exists,
+    GetBlockSizeByte,
+    GetDirectoryStatus,
+    GetFileLocations,
+    GetFileStatus,
+    GetFingerprint,
+    GetSpace,
+    GetStatus,
+    IsDirectory,
+    IsFile,
+    ListStatus,
+    Mkdirs,
+    Open,
+    RenameDirectory,
+    RenameFile,
+    SetOwner,
+    SetMode;
```

Comment:
Should these be in caps? Also this is a fairly hard to maintain list of operations, can we determine whether or not it is a ufs op from the metric name itself instead of having to match this list?

Reply:
I cannot think of a good way of getting the metric names. It is used when the default metric master starts, and the reflection may not work well because there are other methods on the under filesystem interface not accounted as ufs ops.